### DISCUSSION TITLE
How to output cleaner text file?

### DISCUSSION THREAD

**Discussion by ThioJoe** [ggml-org/whisper.cpp#1571] [Q&A]
I use the -otxt option, but the text file it outputs has a space at the beginning of each line, and the transcript is broken up into many lines.

```
 So it all
 looks like this
 and is kind of 
 annoying
```

Any way to make it not do that and just output like a regular transcript?

**Comment by bradmit** [comment]
You could increase the maxlength of the line output perhaps?
